bin/schemats.ts

```diff
@@ -18,10 +18,12 @@ let argv: any = yargs
     .alias('c', 'conn')
     .nargs('c', 1)
     .describe('c', 'database connection string')
-    .demand('t')
```

Comment:
I agree. But then I was thinking if no schema nor tables are provided, we could output all tables from default (public) schema. What do you think?